spec/features/product_category_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'product category dropdown list', js: true do
+  let(:calculator) { create(:calculator) }
+  before do
+    FeatureFlag.get('feature_budget_category').activate
+    visit '/calculator'
+    find(:select, 'product_category')
+    has_select?('product_category', with_options: ['budgetary', 'medium', 'premium'])
```

Comment:
['budgetary', 'medium', 'premium'] should be in const

Reply:
Thanks